.eslintrc.cjs

```diff
@@ -14,6 +14,7 @@ module.exports = {
 			},
 		],
 		'jsdoc/require-jsdoc': 0,
+		'@stylistic/linebreak-style': [2, process.env.NODE_ENV === 'prod' ? 'unix' : 'windows'],
```

Comment:
Please disable this rule instead if it is inconsistent across platforms.

Reply:
I don't quite understand what you mean.🧐

The meaning of this code is that if the platform is unix-like, then use level 2. Or in Windows, use the default value 0. This is a reasonable behavior because in Windows users would end up with `crlf` and everything works fine.

Or if you want users to be forced to use `lf` when submitting their own code, then I think git itself can already do this:

```bash
git config --global core.autocrlf input
```

This is the default value of git's configuration in Windows.

Or if you want users to always keep using `lf` whether checkout or pushing, then you may need to create a file like this:

.gitattributes
```plain
* text=auto eol=lf
```

Reply:
Yes, your last suggestion about .gitattributes looks good to me. Could you do that in your pull request?

Reply:
I just realized that there is something wrong with my description of the situation. There are two situations here:

1. Keeping things as they are, the default behavior of git on Windows is to use `crlf` when pulling and `lf` when pushing. or
2. Add `.gitattribute` file. Git will force all to use `lf` when pulling or pushing.

Reply:
Got it.